security/access_control.rst

```diff
@@ -22,10 +22,10 @@ for each ``access_control`` entry, which determines whether or not a given
 access control should be used on this request. The following ``access_control``
 options are used for matching:
 
-* ``path``
+* ``path`` # a regular expression (without delimiters)
 * ``ip`` or ``ips`` (netmasks are also supported)
-* ``host``
-* ``methods``
+* ``host`` # a regular expression
+* ``methods`` # one or many methods
```

Comment:
The `#` is  bit weird here, since it's just an in-text list (and not a code snippet). I would prefer to do ``` ``path``: a regular expresion (without delimiters)```

Reply:
Done, thanks!